engine/glfw/lib/android/android_init.c

```diff
@@ -203,6 +203,41 @@ GLFWAPI int32_t glfwAndroidWindowOpened()
     return _glfwWin.opened;
 }
 
+GLFWAPI int32_t glfwAndroidVerifySurface()
+{
+    // Although it's the wrong place to do a eglSwapbuffers, we're already handling a bad state from the last opengl error
+    // Verifying the state of the surface is worth it.
+    if (!eglSwapBuffers(_glfwWinAndroid.display, _glfwWinAndroid.surface))
```

Comment:
This function is able to detect if the current window surface is ok.
We use it when checking the OpenGL errors, to decide if we should assert or not.